apps/daimo-mobile/src/view/shared/HistoryList.tsx

```diff
@@ -80,17 +90,37 @@ export function HistoryListSwipe({
     });
     if (month !== lastMonth) {
       stickyIndices.push(rows.length);
-      rows.push(<HeaderRow key={month} title={month} />);
+      rows.push({
+        isHeader: true,
+        month,
+        id: month,
+      });
       lastMonth = month;
     }
-    rows.push(renderRow(t));
+    rows.push({
+      isHeader: false,
+      id: `${t.timestamp}_${t.amount}_${t.blockHash}`,
```

Comment:
recommend using `${t.timestamp}-${t.from}-${t.to}-${t.txHash}-${t.opHash}` as in renderRow , otherwise we'll have edge cases when multiple txs with same amount end up in same block

also you can deduplicate eg with function `getKey(t: Transfer)`